app/models/docker_builder_service.rb

```diff
@@ -71,9 +80,12 @@ def build_image(tmp_dir)
     output.puts("### Running Docker build")
 
     build.docker_image =
-      Docker::Image.build_from_dir(tmp_dir, {}, Docker.connection, registry_credentials) do |output_chunk|
-        handle_output_chunk(output_chunk)
-      end
+      Docker::Image.build_from_dir(
+        tmp_dir,
```

Comment:
true ... useless for bime without

On Wed, Aug 17, 2016 at 11:56 AM, Jon Moter <notifications@github.com>
wrote:

> In app/models/docker_builder_service.rb
> <https://github.com/zendesk/samson/pull/1203#discussion_r75184501>:
>
> > @@ -71,9 +80,12 @@ def build_image(tmp_dir)
> >      output.puts("### Running Docker build")
> >
> >      build.docker_image =
> > -      Docker::Image.build_from_dir(tmp_dir, {}, Docker.connection, registry_credentials) do |output_chunk|
> > -        handle_output_chunk(output_chunk)
> > -      end
> > +      Docker::Image.build_from_dir(
> > +        tmp_dir,
>
> A thought just occurred to me. We might need to specify what the root
> directory is for doing the Docker build.
>
> For example, if there's a repo with a file called $root/scala/Dockerfile,
> there's a decent chance the base directory should be $root/scala, instead
> of $root.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1203/files/efc715ffd4d282b94fec671f6aaa56511202dc73#r75184501>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZ7tJXVWjjOn3je9AA3Ffn89JyynEks5qg1lKgaJpZM4JkwAM>
> .
>